framework/utils/InputData.py

```diff
@@ -453,9 +495,10 @@ def addSub(cls, sub, quantity=Quantity.zero_to_infinity):
     cls.subs.add(sub)
     if cls.subOrder is not None:
       cls.subOrder.append((sub,quantity))
-    elif quantity != Quantity.zero_to_infinity:
-      print("ERROR only zero to infinity is supported if Order==False ",
-            sub.getName()," in ",cls.getName())
+    # I did not see the value of the following check --- wangc
+    #elif quantity != Quantity.zero_to_infinity:
+    #  print("ERROR only zero to infinity is supported if Order==False ",
+    #        sub.getName()," in ",cls.getName())
```

Comment:
@cogljj can you comment on this bit of code from the InputParams algorithms? If it is necessary, can we add a comment block explaining why so we don't question it again? I feel like I questioned it before and ended up leaving it in place.

Reply:
not valid any more. I have removed the commented lines.